pkg/auth/webapp/session_authflow.go

```diff
@@ -774,6 +774,19 @@ func (s *AuthflowScreenWithFlowResponse) Advance(p string, result *Result) {
 	result.RedirectURI = u.String()
 }
 
+// RedirectToFinish is a fix for https://linear.app/authgear/issue/DEV-1793/investigate-sign-in-directly-with-httpsaccountsportalauthgearcom-crash
+// We need Turbo to visit /finish with a full browser redirect,
+// so CSP and connect-src will not kick in.
+func (s *AuthflowScreenWithFlowResponse) RedirectToFinish(route string, result *Result) {
```

Comment:
Should we remove `route` parameter from this function? Because it is always `/authflow/v2/finish`.

Reply:
No, we have v1 finish and v2 finish.

Reply:
ok I see